crow/setup3.py

```diff
@@ -18,15 +18,15 @@
 except:
   eigen_flags = ""
 include_dirs=['include/distributions','include/utilities','contrib/include']
```

Comment:
Possibly, but since windows supports "/" as a directory separator unless we want to support VMS or classic Mac OS or something else exotic, we are safe doing this.